src/Symfony/Bundle/FrameworkBundle/Templating/Helper/FormHelper.php

```diff
@@ -54,7 +54,12 @@ public function getName()
      */
     public function setTheme(FormView $view, $themes)
     {
-        $this->renderer->setTheme($view, $themes);
+        $args = func_get_args();
+        if (!isset($args[2])) {
+            @trigger_error(sprintf('Calling setTheme method of %s without the third boolean argument $useDefaultThemes is deprecated since 3.4 and will be removed in 4.0. Call the method with the third argument included and use boolean true to keep current behaviour.', self::class), E_USER_DEPRECATED);
```

Comment:
I don't understand why we would want to force people to pass a third argument, which should be `true` by default.

Reply:
Hm.. I'm not sure what exactly deprecation message should be then if the third argument is optional? Can't we just use it if it's there, and if not, default to `true`?

Reply:
I think this is the right approach.

Reply:
Okay, will fix it ASAP. Where do you want me to document the fact that in 4.0 the interface is going to receive a third argument? I've seen somewhere in the codebase the usage of an inline comment in the style of `public function setTheme(FormView $view, $themes /* $useDefaultThemes = true */ );`

Reply:
Yes, that's what we are doing.

Reply:
@fabpot Alright, I've updated the code now as per instructions.